components/createNft/FileUploadSection.tsx

```diff
@@ -0,0 +1,211 @@
+import getFileType from '../../utils/getFileType';
+import { NftForm } from '../../utils/Interfaces';
+import FileInformation from '../common/FileInformation';
+import Tooltip from '../common/Tooltip';
+import Button from '../global/Button';
+
+interface FileUploadSectionProps {
+  formData: NftForm;
+  setFormData: Dispatch<SetStateAction<NftForm>>;
+}
+
+const FileUploadSection = function ({
+  formData,
+  setFormData,
+}: FileUploadSectionProps) {
+  const [dragActive, setDragActive] = useState(false);
+  const inputFileRef = useRef<HTMLInputElement>(null);
+  const [viewImage, setViewImage] = useState(false);
+  const [overlayImageSrc, setOverlayImageSrc] = useState('');
+
+  const handleDrag = function (e: DragEvent<HTMLLabelElement>) {
+    e.preventDefault();
+    e.stopPropagation();
+    if (e.type === 'dragenter' || e.type === 'dragover') {
+      setDragActive(true);
+    } else if (e.type === 'dragleave') {
+      setDragActive(false);
+    }
+  };
+  const handleDrop = function (e: DragEvent<HTMLLabelElement>) {
+    e.preventDefault();
+    e.stopPropagation();
+    setDragActive(false);
+    const { files } = e.dataTransfer;
+    if (files && files[0]) {
+      const filesArray = Array.from(files);
+
+      const uploaded = [...formData.nftFiles, ...filesArray];
+
+      setFormData({ ...formData, nftFiles: uploaded });
+    }
+  };
+  const handleChange = function (e: ChangeEvent<HTMLInputElement>) {
+    e.preventDefault();
+    const { files } = e.target;
+    if (files && files[0]) {
+      const filesArray = Array.from(files);
+      const uploaded = [...formData.nftFiles, ...filesArray];
+
+      setFormData({ ...formData, nftFiles: uploaded });
+    }
+  };
+  const onButtonClick = () => {
+    inputFileRef?.current?.click();
+  };
+
+  const deleteFile = (file: File) => {
+    const newFiles = new DataTransfer();
+    const filteredNftFiles = formData.nftFiles.filter(
+      (item: File) => item.name !== file.name,
+    );
+    if (inputFileRef.current !== null) {
+      const files = inputFileRef?.current?.files!;
+      for (let i = 0; i < files.length; i++) {
+        const storedFile = files[i];
+        if (file.name !== storedFile.name) newFiles.items.add(file); // here you exclude the file. thus removing it.
+      }
+      inputFileRef.current.files = newFiles.files;
+    }
+
+    setFormData({ ...formData, nftFiles: filteredNftFiles });
+  };
+  const viewFile = (file: File) => {
+    setViewImage(true);
+    setOverlayImageSrc(URL.createObjectURL(file));
+  };
+  const closeOverlay = () => {
+    setOverlayImageSrc('');
+    setViewImage(false);
+  };
+  return (
+    <>
+      <h4 className="text-lg mb-6 mt-8 font-bold">Upload Files</h4>
+      {viewImage && (
+        <div className="inset-0 flex flex-col items-center bg-gray-900/50 fixed z-10 min-w-full px-[5%] md:px-[15%] pt-20 md:pt-40 min-h-screen">
+          <div>
+            <CrossIcon className=" mb-2 h-10 w-10" onClick={closeOverlay} />
```

Comment:
Add `cursor-pointer` to this element.

![image](https://user-images.githubusercontent.com/40206670/194263045-9e0b73d7-96a6-4a86-8d15-9565a1e5dc1a.png)

Reply:
done.